lib/coralogix.js

```diff
@@ -280,11 +306,11 @@ exports.init = function coralogix_init(startup_time, config, events, log) {
     coralogixApiHost = coralogixConfig.apiHost;
     coralogixPrefix = coralogixConfig.prefix;
     coralogixMappings = new Map(Object.entries(coralogixConfig.mappings));
+    ttlSeconds = coralogixConfig.totalsAccumulatorTtlSeconds ?? 3600;
```

Comment:
My Javascript is not strong - I don't know ??.  || would probably work too.

Reply:
Just googled it, looks fine :-)

Reply:
Yeah it is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing `||` does different thing.